Firmware/Configuration_adv.h

```diff
@@ -289,9 +289,7 @@
   //#define LA_DEBUG_LOGIC     // @wavexx: setup logic channels for isr debugging
 #endif
 
-// Arc interpretation settings:
-#define MM_PER_ARC_SEGMENT 1
-#define N_ARC_CORRECTION 25
+// Arc interpretation settings : Moded to printer default settings (Configuration_prusa.h)
```

Comment:
I would remove this comment entirely (there's also a typo: "moded" -> "moved")

Reply:
I'll wait for this to be resolved before merging